### ISSUE TITLE
Design: Reusable Q&As on Vet Centers

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#15158]
As part of #14790 , Jordan developed a CMS design for adding Q&As onto Vet Center content type. 

The remaining work here is to get that design reviewed by Facilities team to make sure we didn't miss anything.

### Design

- Vet Centers page in [Q&A Figma file](https://www.figma.com/file/MTBtolRaZK4hUkCprXyIiD/Q%26As?type=design&node-id=146%3A2&mode=design&t=HxgOFDiUcS9W0CmZ-1)

## ACs

- [x] CMS Design for Q&As on Vet Centers has been reviewed & approved
- [x] Final design is linked to #10976

**Comment by jilladams** [comment]
@thejordanwood I may have misunderstood the state of things for Vet Centers -- did you end up already doing a layout for that? (IF so, where can I find it?) If not: noted, and we should update ACs to add in: a design has been created.

**Comment by thejordanwood** [comment]
@jilladams Yes, there's a design for this on the Vet Centers page in the [Q&A Figma file](https://www.figma.com/file/MTBtolRaZK4hUkCprXyIiD/Q%26As?type=design&node-id=146%3A2&mode=design&t=HxgOFDiUcS9W0CmZ-1). I've added a link into the ticket body as well.

**Comment by jilladams** [comment]
Thank you! @mmiddaugh @xiongjaneg ^^ Figma link to Q&A on Vet Centers, for review! We can also touch on this in UX refinement today if we have time - will add to the agenda.

**Comment by davidmpickett** [comment]
Does this need to be reviewed in UX refinement today? Not sure what happened last week

**Comment by davidmpickett** [comment]
Is this a Product review or an Engineering review or both?

**Comment by thejordanwood** [comment]
@davidmpickett This is a product review. Last week I showed my initial designs in UX refinement and this week have been working with @aklausmeier to make updates based on feedback that I got. The main update is creating 2 views for this experience – one view for editors that can make changes to this section and another view for editors that can't make changes. 

I think we're in a good place and I can show in UX refinement next week on 9/28. I'll need to run this by an engineer to make sure that we're able to have 2 views for this based on what permissions editors have.
